.github/workflows/prepare-release.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           source_branch: ${{ github.event.inputs.project }}-release-v${{ github.event.inputs.release_version }}
           pr_title: "${{ github.event.inputs.project }} release v${{ github.event.inputs.release_version }}"
           pr_body: "To be merged after the release is complete."
-          pr_label: "kokoro:run"
+          pr_label: "kokoro:run,PR: Merge After Release"
```

Comment:
appears to be a comma separated list: https://github.com/repo-sync/pull-request#advanced-options